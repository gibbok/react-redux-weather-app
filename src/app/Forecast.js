import React from 'react'
import NavigationContainer from '../navigation/NavigationContainer'
import LocationFinderSearchContainer from '../locationFinder/LocationFinderSearchContainer'
import LocationFinderContainer from '../locationFinder/LocationFinderContainer'
import WeatherContainer from '../forecast/weather/WeatherContainer'
import ForecastDailyContainer from '../forecast/forecastDaily/ForecastDailyContainer'
import ForecastHourlyContainer from '../forecast/forecastHourly/ForecastHourlyContainer'
import styles from '../../node_modules/weather-icons/css/weather-icons.css'

const Forecast = () => (
  <div>
    <NavigationContainer />
    <LocationFinderSearchContainer />
    <LocationFinderContainer />
    <WeatherContainer />
    <ForecastDailyContainer />
    <ForecastHourlyContainer />
  </div>
)

export default Forecast
