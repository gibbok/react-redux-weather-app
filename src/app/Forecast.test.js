import React from 'react'
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json'
import Forecast from './Forecast'

describe('<Forecast />', () => {
  it('should render', () => {
    const wrapper = shallow(<Forecast />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})
