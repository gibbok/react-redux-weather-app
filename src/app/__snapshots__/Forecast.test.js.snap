// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Forecast /> should render 1`] = `
<div>
  <Connect(Navigations) />
  <Connect(LocationFinderSearch) />
  <Connect(LocationFinder) />
  <Connect(Weather) />
  <Connect(ForecastDays) />
  <Connect(ForecastHourly) />
</div>
`;
